packages/storybook-react/config/main.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     '@storybook/addon-essentials',
     '@storybook/addon-interactions',
     '@storybook/addon-a11y',
+    '@storybook/addon-docs',
```

Comment:
I think this should be `@storybook/blocks` instead? The `addon-docs` is the one with only version 6 I believe...